Exercise_A/index.html

```diff
@@ -17,11 +17,132 @@
+      <a href="#" class="navbar-brand">THE ONLY BOOTSTRAP PAGE</a>
+      <button class="navbar-toggler ms-auto" data-toggle="collapse" data-target="#navbarMenu">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarMenu">
+        <ul class="navbar-nav ms-auto">
+          <li class="nav-item"><a href='#About' class="nav-link">About</a></li>
+          <li class="nav-item"><a href='#Four' class="nav-link">Four Icons</a></li>
+          <li class="nav-item"><a href='#Bullshit' class="nav-link">Bullshit</a></li>
+          <li class="nav-item"><a href='#Call' class="nav-link">Call Me</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="bg-image p-5" style="background-image: url(./assets/images/header.jpg); height:40rem; background-size: cover; background-repeat: no-repeat; background-position: bottom;">
+      <div class="container">
+        <div class="d-flex justify-content-center">
+          <div class="align-content-center text-center">
+            <h1 class='text-center' style="width:70%; margin-left: 15%; margin-top:10%; color:white;"><strong>HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</strong></h1>
+            <hr class="align-content-center">
+            <p class="lead text-center" style="color:white;"> Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+            <button type="button" class="btn btn-warning">This Button Makes The Page Scrolls</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section style="background-color:orange; padding-bottom: 10%" id="About">
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col-12"><h1 style="color:white; text-align:center; margin-top: 5rem;">Want to make an original website yourself?</h1></div>
+        </div>
+        <hr class="white mt-4">
+        <div class="row justify-content-center text-center mt-4">
+          <div class="col-8"><p1 style="color:white;;">Forget that! Who would ever want to put in all of that effort for a website? Just open up your web browser and type "bootstrap template" into your favorite search engine, like Yahoo! or Bing, and you're on your way! There are hundreds of templates to choose from, but go ahead and pick this same exact template from the first result on google, edit a few lines, and you're on your way! No one will notice!</p1></div>
+        </div>
+      </div>
+    </section>
+
+    <section style="background-color:white; padding-top: 5%; padding-bottom: 5%" id="Four">
+      <div class="row justify-content-center text-center">
+        <div class="col-4 style:"><h1>So fucking creative</h1></div>
+      </div>
+      <hr>
+      <div class="row text">
+        <div class="col-3 d-flex justify-content-center"><i class="bi bi-globe" style="font-size: 50px; color: orange"></i></i></div>

```

Comment:
Hello Anson 
It looks like you forgot to animate the icons on this page, you can make use of the Animate.CSS to add in classes to animate each of these logos